lblod/manage-submission-form-tooling
# Adjust informational text to reduce usage of wrong forms

# Description

DL-5665

This PR adjusts text in alert components in forms :

- Notulen 
- Jaarrekening
- Eindrekening 
- Meerjarenplan(aanpassing) 
- Besluit handhaven na ontvangst schorsingsbesluit 
- Reactie op opvragen bijkomende inlichtingen door de toezichthouder (gemeente/provincie) aan de eredienstbesturen 
- Budget(wijziging)/budgetten – Indiening bij centraal bestuur of representatief orgaan 
- Budget(wijziging)/budgetten – Indiening bij toezichthoudende gemeente of provincie 
- Jaarrekeningen van de besturen van de eredienst
- Meerjarenplannen(wijzigingen) van de besturen van de eredienst
- Advies bij jaarrekening eredienstbestuur
- Besluit over budget(wijziging) eredienstbestuur
- Besluit over meerjarenplan(aanpassing) eredienstbestuur
- Schorsing beslissing eredienstbesturen
- Opvragen bijkomende inlichtingen eredienstbesturen (met als gevolg stuiting termijn)


# Type of change


# Related services

- enrich-submission
- mu-migrations

# How to test 

1. Generate dist file and copy the forms to consuming apps
2. Make sure the following migration has run in Loket 
3. Log as Gemente, Bestuur van de Eredienst, Centrale bestuur van de Eredienst 
4. Save forms, it should be saved without errors.

Altered forms should have: 

Alert with informational text 

e.g:

![alert-box](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/5ca58e29-71d5-43f7-8f15-a97d685aeab4)


Grey box + descriptive text below the file upload field 

e.g : 

![greybox](https://github.com/lblod/manage-submission-form-tooling/assets/38471754/1cae95a4-2c61-49b4-951f-0c5fa33f53a2)


# What to check

- Typos, and form structure  

# Links to other PR's

- N/A

# Notes

- drc restart migrations resource cache enrich-submission
